app_base.htm

```diff
@@ -0,0 +1,55 @@
+<%inherit file="/fomo_app/templates/base.htm" />
+
+<%block name="catalog">
+
+    <div class="content row">
+
+        <!-- column -->
+        <div id="sidebar" class="col-2">
+            <ul class="text-center">
+                <a href="/catalog/index"><li class="sidebar ${'activeSide' if categoryId == 0 else ''}">All Products</li></a>
+                <%! from catalog import models as cmod %>
+                <% categories = cmod.Category.objects.all() %>
+                % for cat in categories:

```

Comment:
Is it necessary to include all of the category objects? Can you just grab the id or other field that you need?

Reply:
OH. You use cat.name in the line below this.

Reply:
I think my code cuts off there so you just can't see the .name. I should change this to "order_by though"